seed/java-sdk/alias/src/main/java/com/seed/alias/core/SeedAliasApiException.java

```diff
@@ -1,12 +1,12 @@
 /**
  * This file was auto-generated by Fern from our API Definition.
  */
-package com.seed.plainText.core;
+package com.seed.alias.core;
 
 /**
  * This exception type will be thrown for any non-2XX API responses.
  */
-public class SeedPlainTextApiError extends SeedPlainTextError {
```

Comment:
are we doing something fancy here to rename `Error` suffixes to `Exception` ?

Reply:
if someone wants to upgrade to 1.x is there a way for them to keep their exception names from breaking ?